src/main/java/frc/robot/constants/robots/RobotA.java

```diff
@@ -39,10 +39,33 @@ public RobotA() {
         visionConstants.LIMELIGHT_ANGLE_OFFSET = 0;
         visionConstants.TARGET_NOT_FOUND_WAIT_TIME = 0.1;
 
+        /* control Constants */
```

Comment:
/* control Constants */
in my opinion, they should be next to their consts:
visionDistanceSettings to visionConst etc.